index.css

```diff
@@ -8,10 +8,16 @@
 }
 
 @media only screen and (max-width: 622px) {
+  .nav {
```

Comment:
no need of css , evry thing can be done with tailwind , try  to remove this file and apply tailwind instead